pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -498,26 +498,25 @@ func (asw *actualStateOfWorld) MarkRemountRequired(
 	defer asw.Unlock()
 	for volumeName, volumeObj := range asw.attachedVolumes {
 		for mountedPodName, podObj := range volumeObj.mountedPods {
-			if mountedPodName != podName {
-				continue
-			}
-
-			volumePlugin, err :=
-				asw.volumePluginMgr.FindPluginBySpec(podObj.volumeSpec)
-			if err != nil || volumePlugin == nil {
-				// Log and continue processing
-				klog.Errorf(
-					"MarkRemountRequired failed to FindPluginBySpec for pod %q (podUid %q) volume: %q (volSpecName: %q)",
-					podObj.podName,
-					podObj.podUID,
-					volumeObj.volumeName,
-					podObj.volumeSpec.Name())
-				continue
-			}
+			if mountedPodName == podName {
```

Comment:
Is this change necessary ?
It seems only indentation changed.

Reply:
en, I will remove it